### ISSUE TITLE
LDS ACK ERROR duplicate listener found

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#15658]
```
2019-07-19T14:47:33.405832Z     warn    ads     ADS:LDS: ACK ERROR 10.28.1.51:38978 sidecar~10.28.1.51~svc04-0-8-745ff67675-xkpjt.service-graph04~service-graph04.svc.cluster.local-140 (svc04-0-8-745ff67675-xkpjt.service-graph04) version_info:"2019-07-19T14:43:20Z/9311" node:<id:"sidecar~10.28.1.51~svc04-0-8-745ff67675-xkpjt.service-graph04~service-graph04.svc.cluster.local" cluster:"svc04-0-8.service-graph04" metadata:<fields:<key:"CONFIG_NAMESPACE" value:<string_value:"service-graph04" > > fields:<key:"INCLUDE_INBOUND_PORTS" value:<string_value:"8080" > > fields:<key:"INTERCEPTION_MODE" value:<string_value:"REDIRECT" > > fields:<key:"ISTIO_META_INSTANCE_IPS" value:<string_value:"10.28.1.51,10.28.1.51,fe80::98ff:44ff:fe51:e1ce" > > fields:<key:"ISTIO_PROXY_SHA" value:<string_value:"istio-proxy:a975561b980463f08689d3debe33bb9eefc80c3d" > > fields:<key:"ISTIO_PROXY_VERSION" value:<string_value:"1.1.3" > > fields:<key:"ISTIO_VERSION" value:<string_value:"1.2.2" > > fields:<key:"POD_NAME" value:<string_value:"svc04-0-8-745ff67675-xkpjt" > > fields:<key:"app" value:<string_value:"svc04-0-8" > > fields:<key:"istio" value:<string_value:"sidecar" > > fields:<key:"pod-template-hash" value:<string_value:"745ff67675" > > fields:<key:"role" value:<string_value:"service" > > fields:<key:"version" value:<string_value:"v1" > > > locality:<region:"us-central1" zone:"us-central1-a" > build_version:"a975561b980463f08689d3debe33bb9eefc80c3d/1.11.0-dev/Clean/RELEASE/BoringSSL" > type_url:"type.googleapis.com/envoy.api.v2.Listener" response_nonce:"c8f56e4f-e79b-485a-ae65-89aa300774d1" error_detail:<code:13 message:"duplicate listener 10.28.1.51_8080 found" >
```

Just based on timing, maybe related to https://github.com/istio/istio/pull/15628

**Comment by howardjohn** [comment]
update: rolled back to before the #15628 pr, I don't think its the cause. I do think this is a regression though

**Comment by howardjohn** [comment]
`git bisect` found 82e30d02b to be the first commit with the issue. fyi @utako

**Comment by howardjohn** [comment]
@utako any progress on this? We may need to revert, this is blocking performance testing for the 1.3 release

**Comment by howardjohn** [comment]
In recent tests of master latest daily I havent been able to reproduce, but I was able to easily in the commit 82e30d0. Its possible we accidentally fixed this -- more investigation needed

**Comment by howardjohn** [comment]
This is fixed as of facd515a8... not sure why @rshriram

**Comment by howardjohn** [comment]
Presumed fixed.. have not been able to reproduce since the commit mentioned above, and I could 100% of the time before
